### PR TITLE
feat(webapp): create event design optimization

### LINKED ISSUE #6266: 🚀 [Feature] Webapp: Event create design optimization
## 🚀 Feature

On the page create event we need some design optimization. 
- The tab selection is not optimal
- The date-picker could be grayed out, mark the current date, 
- Start date has no default

## 🤖 Todo
- [ ] datepicker (ausgrauen, markein aktuelles Datum)
- [ ] Auswahl zwischen Post/Event nicht schön
- [ ] Required feld für Start Zeitpunkt enhält keinen Text

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
- tab post
- datepicker, hide before today, hide seconds, minute steps 15 min
 

### Issues
- fixes #6266 

Bildschirmfoto vom 2023-04-26 13-45-28

Bildschirmfoto vom 2023-04-26 13-45-45

### CODE REVIEW

**Comment by ogerly** [comment]
Bildschirmfoto vom 2023-04-26 14-55-52
Bildschirmfoto vom 2023-04-26 14-55-25
Bildschirmfoto vom 2023-04-26 14-55-03

File: webapp/graphql/PostMutations.js [hunk 1]
```diff
@@ -4,27 +4,43 @@ export default () => {
   return {
     CreatePost: gql`
       mutation (
+        $id: ID
         $title: String!
+        $slug: String
         $content: String!
         $categoryIds: [ID]
-        $image: ImageInput
```
Comment (Elweyn) [inline webapp/graphql/PostMutations.js:10]: why did this get removed?

File: webapp/graphql/PostMutations.js [hunk 2]
```diff
@@ -4,27 +4,43 @@ export default () => {
...
+        $postType: PostType
+        $eventInput: _EventInput
       ) {
         CreatePost(
+          id: $id
           title: $title
+          slug: $slug
           content: $content
           categoryIds: $categoryIds
-          image: $image
```
Comment (Elweyn) [inline webapp/graphql/PostMutations.js:17]: This also got removed somehow

**Review by Elweyn — CHANGES_REQUESTED** [review]

File: webapp/graphql/PostMutations.js (hunk 1 above)
Comment (Elweyn) [inline webapp/graphql/PostMutations.js:10]: Got deleted somewhere on the process but think is still needed

File: webapp/graphql/PostMutations.js (hunk 2 above)
Comment (Elweyn) [inline webapp/graphql/PostMutations.js:17]: Got deleted somewhere on the process but think is still needed

**Review by Elweyn — APPROVED** [review]
This view of the event location and dates are good, maybe we want it in another place of the teaser but we would make this in a new issue

**Pull request merged**